playbook/app/pb_kits/playbook/pb_table/table.html.erb

```diff
@@ -6,21 +6,11 @@
 
 <%= content_tag(:div, class: responsive_class, 'data-pb-table-wrapper' => true) do %>
   <% if object.tag == "table" %>
-    <%= content_tag(:table,
```

Comment:
I think we also need to tackle the subcomponents of the table kit on this story, table_body, table_cell, table_header, table_head, table_row